recipes/ocw/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "ocw" %}
+{% set version = "1.1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "8f12405ec02ea327b1e0cf65547e409232f67577be6eb348d2ea9011827e2c4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy >=1.10.4
+    - scipy >=0.17.0
+    - matplotlib >=1.5.1
+    - basemap >=1.0.7
+    - netcdf4 >=1.2.2
+    - h5py >=2.6.0
+    - bottle >=0.12.9
+    - pydap >=3.1.1
+    - podaacpy >=1.0.2
+    - python-dateutil >=2.5.3
+    - mock >=2.0.0
+    - myproxyclient >=1.4.3
+    - webtest >=2.0.21
+    - esgf-pyclient >=0.1.6
+
+test:
+  imports:
+    - ocw
+    - ocw.data_source
+    - ocw.esgf
+
+about:
+  home: http://climate.apache.org/
+  license: Apache License, Version 2.0
```

Comment:
So we at conda-forge prefer the shortened license form. So `Apache 2.0` or similar.